target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -72,6 +72,17 @@ define Device/ubnt-xm
   KERNEL := kernel-bin | append-dtb | relocate-kernel | lzma | uImage lzma
 endef
 
+define Device/ubnt-sw
+  $(Device/ubnt)
+  DEVICE_VARIANT := SW
+  DEVICE_PACKAGES += kmod-usb-ohci
+  IMAGE_SIZE := 7552k
+  UBNT_TYPE := SW
+  UBNT_CHIP := ar7240
```

Comment:
Please set up UBNT_BOARD and UBNT_VERSION in this node as done for the other nodes in:
https://patchwork.ozlabs.org/patch/1205037/

Reply:
Done.